src/helpers/errors.ts

```diff
@@ -14,25 +13,26 @@ const APP_CONFIG_URL = 'https://app.contentful.com/deeplink?link=apps';
 * message's "link" url and title.
 
 * E.g.
-* `Return to $${APP_CONFIG_URL}|App Config Page$ to enter a valid API key`
+* `Return to $${APP_CONFIG_URL}|Configuration Page$ to enter a valid API key`
 * Will be parsed as: 
 * `Return to https://app.contentful.com/deeplink?link=apps to enter a valid
 * API key`.
 */
 
 const ERROR_MESSAGES = {
   InvalidApiKeyError: {
-    message: `Return to $${APP_CONFIG_URL}|App Config Page$ to enter a valid API key`,
+    message: `Return to the Configuration Page to enter a valid API Key or 
+    contact your system administrator.`,
     name: 'Invalid API Key',
     type: 'negative',
   },
   NoSourcesError: {
-    message: `Go to $${DASHBOARD_URL}$ to add an imgix source.`,
-    name: 'You have no imgix sources',
+    message: `Go to $${DASHBOARD_URL + '/sources'}$ to add an imgix Source.`,
+    name: 'You have no imgix Sources',
     type: 'warning',
   },
   NoOriginImagesError: {
-    message: `Go to $${DASHBOARD_URL}$ to add an imgix source.`,
+    message: `Go to $${DASHBOARD_URL}$ to add Origin images to this Source.`,
```

Comment:
Might want to double check me on this, but in the feedback I think we said to link to the sources page in this error as well:
```suggestion
    message: `Go to $${DASHBOARD_URL + '/sources'}$ to add Origin images to this Source.`,
```